.github/workflows/purge-test-resources.yaml

```diff
@@ -22,13 +23,13 @@ jobs:
       - name: List Test Resource Groups
         run: |
           echo "## Test resource group list" >> $GITHUB_STEP_SUMMARY
-          az group list --query "[?starts_with(name, 'samplestest-')].{Name:name, creationTime:tags.creationTime}" -o json > resource_groups.json
+          az group list --query "[?starts_with(name, 'rg-samplestest-')].{Name:name, creationTime:tags.creationTime}" -o json > resource_groups.json
```

Comment:
What is causing this to change?

Reply:
I think it was wrong before. We can see that the resource groups we create have a prefix of `rg-samplestest`: https://github.com/radius-project/samples/blob/v0.33/.github/workflows/test.yaml#L122.